RAW/ViewController.swift

```diff
@@ -0,0 +1,64 @@
+//
+//  ViewController.swift
+//  RAW
+//
+//  Created by Bharat malhotra on 13/07/18.
+//  Copyright © 2018 Bharat malhotra. All rights reserved.
+//
+
+import UIKit
+
+class ViewController: UIViewController {
+
+    @IBOutlet weak var tableView: UITableView!
+    var posts : [PostModel] = []
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        // Do any additional setup after loading the view, typically from a nib.
+        fetchDataWithAPI()
+    }
+    
+    func fetchDataWithAPI(){
+        WebService().fetchRawData(sucess:{[weak self](userData) in
```

Comment:
create a protocol of the web service and inject it into the view controller. Ask me if that doesn't make sense.